client/src/lib/repositories/tags.ts

```diff
@@ -0,0 +1,14 @@
+import type { Fetch } from "src/definitions/fetch";
+import type { Tag } from "src/definitions/tag";
+import { getApiUrl, getHeaders } from "../fetch";
+
+type GetTags = (opts: { fetch: Fetch; apiToken: string }) => Promise<Tag[]>;
+
+export const getTags: GetTags = async ({ fetch, apiToken }) => {
+  const url = `${getApiUrl()}/tags`;
+  const request = new Request(url, {
+    headers: getHeaders(apiToken),
+  });
+  const response = await fetch(request);
+  return response.json();
+};
```

Comment:
Brancher la nouvelle logique d'interception des requêtes à l'API :

* Import de `makeApiRequest`
* Puis :

```suggestion
type GetTags = (opts: { fetch: Fetch; apiToken: string }) => Promise<Maybe<Tag[]>>;

export const getTags: GetTags = async ({ fetch, apiToken }) => {
  const url = `${getApiUrl()}/tags`;
  const request = new Request(url, {
    headers: getHeaders(apiToken),
  });
  const response = await makeApiRequest(fetch, request);
  return Maybe.map(response, response => response.json());
};
```